internal/compiler/langtype.rs

```diff
@@ -805,21 +832,32 @@ pub fn unit_product_length_conversion(a: &[(Unit, i8)], b: &[(Unit, i8)]) -> Opt
     }
 }
 
+pub fn unit_product_length_conversion_px_to_phx(a: &[(Unit, i8)], b: &[(Unit, i8)]) -> Option<i8> {
+    unit_product_length_conversion(a, b, Unit::Px, Unit::Phx)
+}
+
+pub fn unit_product_length_conversion_rem_to_px(a: &[(Unit, i8)], b: &[(Unit, i8)]) -> Option<i8> {
+    unit_product_length_conversion(a, b, Unit::Rem, Unit::Px)
+}
```

Comment:
I'm not sure why this function had to be duplicated.

Reply:
Given the many call site it's more compact. But if you prefer I remove these "overloads" and pass the parameters on each call site. Let me know what you prefer.

Reply:
I wonder why we need these extra argument arguments at all.

Reply:
Also what about rem_to_phx?

Reply:
What i mean i that unit_product_length_conversion should work with all units and there is no need to pass these values.

Reply:
Ok, that would be even nicer - I see. I'm not entirely sure yet how but I'll give it a try :)

Reply:
Ok, I've changed just `unit_product_length_conversion` and added a `rem` to `phx` conversion. I hope I didn't missing anything, it's still a little egg-shell mode in that function for me :)

Reply:
(Could also do the inverse, but not sure that truly makes sense)